Sources/SnapshotTesting/Strategy/NSView.swift

```diff
@@ -0,0 +1,43 @@
+#if os(macOS)
+import Cocoa
+
+extension Strategy where Snapshottable == NSView, Format == NSImage {
```

Comment:
I've reorganized a bit with `NSView` and `UIView`-specific files, while `View` contains some shared view code.